### ISSUE TITLE
Autoload failing because classes are in the same file

### ISSUE THREAD

**Issue by tiagobutzke** [konduto/php-sdk#1]
Hi guys, I found an issue here.

```
Class 'Konduto\Models\BusTravelLeg' not found in /var/www/brazil/vendor/konduto/sdk/src/Models/Model.php
```

When it tries instantiate the BusTravelLeg, it doesn't works, because the autoload is seeking for a BusTravelLeg file to be included.

Regards,

**Comment by adri0** [comment]
Thank you for reporting it, tiagobutzke. I've just fixed it, along with other bugs, and released v1.3.1

**Comment by tiagobutzke** [comment]
Thank you @adri0! :)
